board_types.txt

```diff
@@ -64,6 +64,8 @@ Reserved "NXP ucans32k146"             34
 Reserved "NXP FMURT1170-V1"            35 # AKA PX4 FMU V6XRT
 Reserved "NXP Tropic"                  36
 
+Reserved “ZEROONE_X6"                 101
```

Comment:
We only want one entry - so delete this

Reply:
Do you mean we only need to add the following sentences?
#IDs 5600-5699 reserved for ZeroOne
AP_HW_ZeroOne_X6 5600
AP_HW_ZeroOne_PMU 5601
AP_HW_ZeroOne_GNSS 5602

Reply:
Ok,I have removed this line:
Reserved “ZEROONE_X6"                 101

Reply:
@davids5 Do you think the modified version is OK?